source/reference/modules/deployer/index.rst

```diff
@@ -355,6 +354,122 @@ the steps described in :ref:`debugging-deployer-issues`
   Changing or deleting a processed commit file could cause unchanged files to be indexed again and
   it should be done as a last resort in case of errors.
 
+^^^^^^^^^^^^^^^^^
+Document Indexing
+^^^^^^^^^^^^^^^^^
+Crafter Deployer indexes documents via the following configured items:
+
+- A full text index of any document that has a mimetype that matches the configured list of mimetypes.
+- Indexing of any remote document that matches the configured list of remote documents pattern
+- Indexing of jacketed documents with anything that matches the configured pattern.
+
```

Comment:
- Indexing of filenames and meta-data for other content items. [Check what we do with binary files like .jpg, or similar where we have meta-data but not textual content and explain it here.]

Reply:
The above is done for authoring. I don't think it's done for delivery. Check that, and if it's only delivery, remove it from this.